Controls/SerialOutputCoT.cs

```diff
@@ -269,6 +283,11 @@ string FindUIDviaSysid(byte sysid)
             return "NOsysid" + sysid;
         }
 
+        bool isValidStr(object obj)
+        {
+            return (obj != null && obj.ToString().Length > 0);

```

Comment:
https://learn.microsoft.com/en-us/dotnet/api/system.string.isnullorempty?view=net-8.0

Reply:
Thanks for the tip. I was just refactoring your code though :)

`if (callsign != null && callsign.ToString().Length > 0 && endpoint != null && endpoint.ToString().Length > 0`